cmd/minikube/cmd/start.go

```diff
@@ -350,6 +350,17 @@ func validateConfig() {
 		exit.Usage("Sorry, the --hidden feature is currently only supported with --vm-driver=kvm2")
 	}
 
+	// Adding extra option (resolv-conf) for none driver on ubutnu to address /issues/3511
+	if viper.GetString(vmDriver) == constants.DriverNone && noneNeedsResolveConf() {
+		o := "kubelet.resolv-conf=/run/systemd/resolve/resolv.conf"
+		err := extraOptions.Set(o)
+		if err == nil {
+			glog.Infof("Sucsessfully set extra option (%s) for none driver on ubuntu-like OS.", o)
```

Comment:
Spelling: "Successfully". Also, no need to mention ubuntu-like OS. If you need to, it should be capitalized: "Ubuntu".